### PR TITLE
5.x.x/media input requests

### PR DESCRIPTION
**Pull request by Pjiesco**

### CODE REVIEW

File: src/main/java/io/obswebsocket/community/client/OBSRemoteController.java [hunk 1]
```diff
@@ -9,6 +9,7 @@
 import io.obswebsocket.community.client.message.request.filters.*;
 import io.obswebsocket.community.client.message.request.general.*;
 import io.obswebsocket.community.client.message.request.inputs.*;
+import io.obswebsocket.community.client.message.request.mediaInputs.*;
```
Comment (ChristopheCVB) [inline src/main/java/io/obswebsocket/community/client/OBSRemoteController.java:12]: change package name to lowercase

File: src/main/java/io/obswebsocket/community/client/message/response/mediaInputs/GetMediaInputStatusResponse.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+    @Getter
+    @ToString
+    public static class Data {
+        private MediaState mediaState;
+        private Long mediaDuration; // optional
+        private Long mediaTimestamp; // optional
+    }
+
+    public enum MediaState {
```
Comment (ChristopheCVB) [inline src/main/java/io/obswebsocket/community/client/message/response/mediaInputs/GetMediaInputStatusResponse.java:26]: I'm not sure if this should stay in this class or be moved to the `model` package.
@TinaTiel what do you think?

File: src/main/java/io/obswebsocket/community/client/message/response/mediaInputs/NextMediaInputPlaylistItemResponse.java [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+import lombok.Getter;
+import lombok.ToString;
+
+@Getter
+@ToString(callSuper = true)
+public class NextMediaInputPlaylistItemResponse extends RequestResponse {
+    public NextMediaInputPlaylistItemResponse() {
+        super(Request.Type.NextMediaInputPlaylistItem);
+    }
+}
```
Comment (ChristopheCVB) [inline src/main/java/io/obswebsocket/community/client/message/response/mediaInputs/NextMediaInputPlaylistItemResponse.java:14]: add line break

File: src/main/java/io/obswebsocket/community/client/message/response/mediaInputs/GetMediaInputStatusResponse.java [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+        @SerializedName("stopped")
+        STOPPED,
+        @SerializedName("ended")
+        ENDED,
+        @SerializedName("error")
+        ERROR,
+        @SerializedName("unknown")
+        UNKNOWN
+    }
+}
```
Comment (ChristopheCVB) [inline src/main/java/io/obswebsocket/community/client/message/response/mediaInputs/GetMediaInputStatusResponse.java:46]: add line break

File: src/main/java/io/obswebsocket/community/client/message/request/mediaInputs/StopMediaInputRequest.java [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+public class StopMediaInputRequest extends MediaInputRequest {
+    private final Data requestData;
+
+    @Builder
+    private StopMediaInputRequest(String inputName) {
+        super(Type.StopMediaInput);
+
+        this.requestData = Data.builder().inputName(inputName).build();
+    }
+}
```
Comment (ChristopheCVB) [inline src/main/java/io/obswebsocket/community/client/message/request/mediaInputs/StopMediaInputRequest.java:18]: add line break

File: src/main/java/io/obswebsocket/community/client/message/request/mediaInputs/SetMediaInputTimecodeRequest.java [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+    }
+
+    @Getter
+    @ToString(callSuper = true)
+    @SuperBuilder
+    static class Data extends MediaInputRequest.Data {
+        @NonNull
+        private final Long mediaTimestamp;
+    }
+}
```
Comment (ChristopheCVB) [inline src/main/java/io/obswebsocket/community/client/message/request/mediaInputs/SetMediaInputTimecodeRequest.java:28]: add line break

File: src/main/java/io/obswebsocket/community/client/message/request/mediaInputs/SetMediaInputPauseStateRequest.java [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+    }
+
+    @Getter
+    @ToString(callSuper = true)
+    @SuperBuilder
+    static class Data extends MediaInputRequest.Data {
+        @NonNull
+        private final Boolean pause;
+    }
+}
```
Comment (ChristopheCVB) [inline src/main/java/io/obswebsocket/community/client/message/request/mediaInputs/SetMediaInputPauseStateRequest.java:28]: add line break

File: src/main/java/io/obswebsocket/community/client/message/request/mediaInputs/RestartMediaInputRequest.java [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+public class RestartMediaInputRequest extends MediaInputRequest {
+    private final Data requestData;
+
+    @Builder
+    private RestartMediaInputRequest(String inputName) {
+        super(Type.RestartMediaInput);
+
+        this.requestData = Data.builder().inputName(inputName).build();
+    }
+}
```
Comment (ChristopheCVB) [inline src/main/java/io/obswebsocket/community/client/message/request/mediaInputs/RestartMediaInputRequest.java:18]: add line break

File: src/main/java/io/obswebsocket/community/client/message/request/mediaInputs/PreviousMediaInputPlaylistItemRequest.java [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+public class PreviousMediaInputPlaylistItemRequest extends MediaInputRequest {
+    private final Data requestData;
+
+    @Builder
+    private PreviousMediaInputPlaylistItemRequest(String inputName) {
+        super(Type.PreviousMediaInputPlaylistItem);
+
+        this.requestData = Data.builder().inputName(inputName).build();
+    }
+}
```
Comment (ChristopheCVB) [inline src/main/java/io/obswebsocket/community/client/message/request/mediaInputs/PreviousMediaInputPlaylistItemRequest.java:18]: add line break

File: src/main/java/io/obswebsocket/community/client/message/request/mediaInputs/OffsetMediaInputTimecodeRequest.java [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+    }
+
+    @Getter
+    @ToString(callSuper = true)
+    @SuperBuilder
+    static class Data extends MediaInputRequest.Data {
+        @NonNull
+        private final Long timestampOffset;
+    }
+}
```
Comment (ChristopheCVB) [inline src/main/java/io/obswebsocket/community/client/message/request/mediaInputs/OffsetMediaInputTimecodeRequest.java:28]: add line break

**Review by ChristopheCVB — CHANGES_REQUESTED** [review]
Add line breaks on files lacking it.

File: src/main/java/io/obswebsocket/community/client/message/response/mediaInputs/GetMediaInputStatusResponse.java (hunk 2 above)
Comment (TinaTiel) [reply src/main/java/io/obswebsocket/community/client/message/response/mediaInputs/GetMediaInputStatusResponse.java:26]: Nah, probably fine as-is for now.
It's only in 2 responses, and they differ on whether mediaTimestamp is optional.

Comment (ChristopheCVB) [reply src/main/java/io/obswebsocket/community/client/message/response/mediaInputs/GetMediaInputStatusResponse.java:26]: I was talking about the enum there

**Review by TinaTiel — APPROVED** [review]
Approving, pending the change to package name as noted by Christophe

File: src/main/java/io/obswebsocket/community/client/message/response/mediaInputs/GetMediaInputStatusResponse.java (hunk 2 above)
Comment (TinaTiel) [reply src/main/java/io/obswebsocket/community/client/message/response/mediaInputs/GetMediaInputStatusResponse.java:26]: OH...My mistake. 
Coincidentally, my response is the same; so far MediaState is only used in this one response so it makes sense to keep it here for now.

**Review by ChristopheCVB — APPROVED** [review]

**Pull request merged**